controllers/reconcilers/configuration/prometheus.go

```diff
@@ -478,7 +478,7 @@ func (r *Reconciler) reconcilePrometheus(ctx context.Context, cr *v1.Observabili
 	return nil
 }
 
-//resource.Quantity should be greater than than the default 250Gi
+//resource.Quantity should be greater than the default 250Gi
 func validateOverridePrometheusStorageSize(q resource.Quantity, cr *v1.Observability) bool {
```

Comment:
As far as i can tell this validate function checks if the value is above 0 although the comment indicates otherwise. 🤔

Reply:
The function should compare current storage quantity with override quantity (q). `CmpInt64` returns -1, 0, 1 based on comparison. Should have return of true if `CmpInt64` is `>= 0`

Reply:
oh. haha. should have read it properly. 🤦‍♀️